Changes to be able to build as Java 11 project

obfischer opened the pull request:
Changes to the project to be able to run it as Java 11 application.+ deps for 11

Commit:
Changes to be able to build as Java 11 project

Changes to the project to be able to run it as Java 11 application.+ deps for 11

StephanPirnbaum commented on pom.xml:
```diff
@@ -522,9 +480,19 @@
                 <scope>test</scope>
             </dependency>
+
+            <dependency>
```
Should be removed, was only added for jaxrs plugin, which is archived now.

obfischer replied:
Do you refer to the Mockito dependency?

StephanPirnbaum commented on pom.xml:
```diff
@@ -536,12 +504,21 @@
             <version>${jqa-neo4jbackend.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Why was this added?

DirkMahler committed:
set development version to 2.0.0-SNAPSHOT

DirkMahler commented on pom.xml:
```diff
@@ -522,9 +480,27 @@
+                <scope>provided</scope>
+            </dependency>
+
+            <dependency>
```
This dependency has exclusively (and not intentionally) been used by the JAX-RS plugin which is no longer part of the core distribution, can be removed.

obfischer replied:
Removed, commited and pushed.

Commit:
Dependency javax.xml.ws:jaxws-api removed

These dependency has been removed, as the JAX-RS plugin is not longer
part of the core distribution.
